### DISCUSSION TITLE
Mixed Charts

### DISCUSSION THREAD

**Discussion by sprout-apps** [patrykandpatrick/vico#282] [Questions] [answered]
I need to implement a growth chart, similar to what you get at the doctor office. There are several percentile lines displayed as as a multi-line chart. Then over the growth percentiles I need to display a scatter plot of the actual child measurements. Is it possible to do this?

**Comment by sprout-apps** [comment, marked as answer]
Looks like this can be achieved with my data line having LineChart.LineSpec with zero thickness and by adding a point specification to it.

**Reply by patrickmichalik** [reply]
Hello! That’s right—Vico doesn’t have a dedicated chart type for scatter charts, but you can use a `LineChart` with an invisible line. I’m glad that you’ve found a solution, and since you have, I’ll be closing this discussion. We’ll look into implementing a check in `LineChart` that will prevent redundant calculations from being run when the line is invisible.
